pkg/tfbridge/tokens_test.go

```diff
@@ -699,8 +755,13 @@ func TestMaxItemsOneAliasing(t *testing.T) {
 
 	autoAliasing(info, metadata)
 	assert.False(t, *info.Resources["pkg_r1"].Fields["f1"].MaxItemsOne)
-	assert.False(t, *info.Resources["pkg_r1"].Fields["f2"].MaxItemsOne)
-	assert.Equal(t, `{
+
+	f2Schema := info.P.ResourcesMap().Get("pkg_r1").Schema().Get("f2")
+	f2Overrides := info.Resources["pkg_r1"].Fields["f2"]
+
+	assert.False(t, tfbridge.IsMaxItemsOne(f2Schema, f2Overrides))
```

Comment:
The new implementation does not inject MaxItemsOne override here because it's not necessary, as evidenced by the new assert, without the override the system still decides on False.